core/src/main/shim/3.0/SparkShim.scala

```diff
@@ -41,13 +43,17 @@ object SparkShim extends SparkShimBase {
       examples: String,
       note: String,
       since: String): ExpressionInfo = {
-    // TODO fix this up later.
     new ExpressionInfo(
       className,
       db,
       name,
       usage,
-      arguments
+      arguments,
+      examples,
+      note,
+      "", // group
```

Comment:
We have `group` defined in `functions.yml`, can we add it here?

Reply:
The constructor verifies that `group` fall into one of Spark's valid group categories (https://github.com/apache/spark/blob/c8f3bd861d96cf3f7b01cd9f864c181a57e1c77a/sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionInfo.java#L139), which don't match with our groups.